Main.cpp

```diff
@@ -133,27 +133,27 @@ ScenarioGUI *g_pScenarioGUI = 0;
 Controller *g_pMainMenuController = 0;
 
 enum StarSize {
-    StarSmall = 0,
-    StarLarge,
-    StarHuge,
+	StarSmall = 0,
+	StarLarge,
+	StarHuge,
 };
 
 struct Star {
-    // Bitmap representation
-    BITMAP *m_pBitmap;
-    // Center location on screen
-    Vector m_Pos;
-    // Bitmap offset
-//    int m_Offset;
-    // Scrolling ratio
-    float m_ScrollRatio;
-    // Normalized intensity 0-1.0
-    float m_Intensity;
-    // Type
-    StarSize m_Size;
-
-    Star() { m_pBitmap = 0; m_Pos.Reset(); m_ScrollRatio = 1.0; m_Intensity = 1.0; m_Size = StarSmall; }
-    Star(BITMAP *pBitmap, Vector &pos, float scrollRatio, float intensity) { m_pBitmap = pBitmap; m_Pos = pos; m_ScrollRatio = scrollRatio; m_Intensity = intensity; }
+	BITMAP *m_Bitmap = nullptr;
+	int m_PosX = 0;
+	int m_PosY = 0;
+	float m_ScrollRatio = 1.0F;
+	int m_Intensity = 0; //!< Intensity value on a scale from 0 to 255.
+	StarSize m_Size = StarSmall;
+
+	Star() {}
+	Star(BITMAP *pBitmap, int posX, int posY, float scrollRatio, int intensity) :
```

Comment:
Just did a quick test, looks like you don't actually need this constructor at all because it isn't being used and unlikely to be, so you can remove it entirely.
You can remove `Star() {}` as well, a default constructor will be created by the compiler.

Reply:
Right, I just thought it's nice to have. But sure, I'll remove them.